polaris/dataset/converters/_pdb.py

```diff
@@ -0,0 +1,158 @@
+from typing import TYPE_CHECKING, Optional, Sequence, List, Union
+
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import fastpdb
+
+from polaris.dataset import ColumnAnnotation, Modality
+from polaris.dataset._adapters import Adapter
+from polaris.dataset.converters._base import Converter, FactoryProduct
+
+if TYPE_CHECKING:
+    from polaris.dataset import DatasetFactory
+
+
+DTYPE_DICT = {
+    "chain_id": "U4",
+    "res_id": int,
+    "ins_code": "U1",
+    "res_name": "U5",
+    "hetero": bool,
+    "atom_name": "U6",
+    "element": "U2",
+}
+
+KEYS = [
+    "X",
+    "Y",
+    "Z",
+    "chain_id",
+    "res_id",
+    "ins_code",
+    "res_name",
+    "hetero",
+    "atom_name",
+    "element",
+    "atom_id",
+    "b_factor",
+    "occupancy",
+    "charge",
+]
+
+
+class PDBConverter(Converter):
+    """
+    Converts PDB files into a Polaris dataset.
+    During the conversion, only the most essential structural information is retained, including 3D coordinates, chain ID, residue ID, insertion code, residue name, heteroatom indicator, atom name, element, atom ID, B-factor, occupancy, and charge.
```

Comment:
> only the most essential structural information is retained

**Question:** What information do we _lose_? It sounds pretty complete to me! Wasn't this lossless?